Quaver.Shared/Screens/Select/UI/Modifiers/DrawableModifierBool.cs

```diff
@@ -31,7 +32,15 @@ public override List<DrawableModifierOption> CreateModsDialogOptions()
                 new DrawableModifierOption(this, "ON", (o, e) =>
                 {
                     if (!ModManager.Mods.HasFlag(Modifier.ModIdentifier))
+                    {
                         ModManager.AddMod(Modifier.ModIdentifier);
+                        if (Modifier.Name == "No Slider Velocities"
+                            || Modifier.Name == "No Fail"
+                            || Modifier.Name == "Autoplay")
+                        {
+                            NotificationManager.Show(NotificationLevel.Warning, "WARNING! Using this mod combination will not submit your score for ranked!");
```

Comment:
If we're using a notification, show the name of the unranked modifier(s) rather than saying "this mod combination".